pkg/cmd/list.go

```diff
@@ -396,8 +399,8 @@ func (w *WhoCan) getRolesFor(action resolvedAction) (roles, error) {
 }
 
 // GetClusterRolesFor returns a set of names of ClusterRoles matching the specified Action.
-func (w *WhoCan) getClusterRolesFor(action resolvedAction) (clusterRoles, error) {
-	crl, err := w.clientRBAC.ClusterRoles().List(meta.ListOptions{})
+func (w *WhoCan) getClusterRolesFor(ctx context.Context, action resolvedAction) (clusterRoles, error) {
```

Comment:
Don't change the signature of this method. Pass context.Background() to the client